apps/e2e/cypress/e2e/back-office-appeals/assignUserToCase.spec.js

```diff
@@ -0,0 +1,94 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { AppealsListPage } from '../../page_objects/appealsListPage';
+import { SearchResultsPage } from '../../page_objects/searchResultsPage';
+import { projectInformation } from '../../support/utils/createProjectInformation';
+
+const page = new AppealsListPage();
+describe('Appeals feature', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('Case officer should be able to assign themselves to a case using name search', () => {
+		cy.visit('/appeals-service/appeals-list');
+		page.clickAppealFromList(18);
+		page.clickCaseOfficer(15);
+		page.nationalListSearch('case');
+		page.basePageElements.summaryListValue('caseteamofficer.test@planninginspectorate.gov.uk');
+		page.clickLinkByText('Assign');
+		page.selectRadioButtonByValue('Yes');
+		page.clickButtonByText('Continue');
+		page.validateBannerMessage('Case officer has been assigned');
+		page.checkAnswerSummaryValue('caseteamofficer.test@planninginspectorate.gov.uk');
+	});
+
+	it('Inspector should be able to assign themselves to a case using name search', () => {
+		cy.visit('/appeals-service/appeals-list');
+		page.clickAppealFromList(18);
+		page.clickInspector(15);
+		page.nationalListSearch('test');
+		page.basePageElements.summaryListValue('inspectorappeals.test@planninginspectorate.gov.uk');
+		page.clickLinkByText('Assign');
+		page.selectRadioButtonByValue('Yes');
+		page.clickButtonByText('Continue');
+		page.validateBannerMessage('Inspector has been assigned');
+		page.checkAnswerSummaryValue('inspectorappeals.test@planninginspectorate.gov.uk');
+	});
+
+	it('Case officer should be able to change assigned user', () => {
+		cy.visit('/appeals-service/appeals-list');
+		page.clickAppealFromList(18);
+		page.clickCaseOfficer(16);
+		page.nationalListSearch('Rachel');
+		page.basePageElements.summaryListValue('rachel.harvey@planninginspectorate.gov.uk');
+		page.clickLinkByText('Assign');
+		page.selectRadioButtonByValue('Yes');
+		page.clickButtonByText('Continue');
+		page.validateBannerMessage('Case officer has been assigned');
+		page.checkAnswerSummaryValue('Rachel HarveyRachel.Harvey@planninginspectorate.gov.uk');
```

Comment:
I may not be understanding this correctly but this doesn't mirror the email structure on lines 25 and 38. You have uppercase (where you do not have it on line 46. And you also have a name in the value. Is this expected behaviour?

Reply:
thanks, this is due to the caseofficer email having a blank name - when the name has been added I can amend so that it will display the same as my name and email (51). The uppercase vs not uppercase i can amend - my name and email does always have initials capitalised, so I could change this for line 46. It seems to not take case into consideration when assigning email.  For the caseofficer/inspector emails it's not capitalised anywhere (i think this is due to how it has been created in Ad groups).